tests/foreman/ui/test_http_proxy.py

```diff
@@ -244,6 +246,59 @@ def test_set_default_http_proxy(module_org, module_location, setting_update, tar
         assert result['table'][0]['Value'] == "Empty"
 
 
+@pytest.mark.tier1
+@pytest.mark.run_in_one_thread
+@pytest.mark.parametrize('setting_update', ['content_default_http_proxy'], indirect=True)
+def test_positive_set_default_http_proxy(
+    request, module_org, module_location, setting_update, target_sat
+):
+    """Setting "Default HTTP proxy" when new HTTP proxy is created.
+
+    :id: e93733e1-5c05-4b7f-89e4-253b9ce55a5b
+
+    :steps:
+        1. Navigate to Infrastructure > Http Proxies
+        2. Create a Http Proxy and set "Default content HTTP proxy"
+        3. Navigate to Administer > Settings > Content tab
+        4. Verify the "Default HTTP Proxy" setting with created above.
+        5. Update "Default HTTP Proxy" to "no global default".
+
+    :Verifies: SAT-28860
+
+    :expectedresults: Creating Http Proxy with option "Default content HTTP proxy",
+        updates setting "Default HTTP Proxy" succesfully.
+    """
+    property_name = setting_update.name
+    http_proxy_name = gen_string('alpha', 15)
+    http_proxy_url = settings.http_proxy.un_auth_proxy_url
+
+    with target_sat.ui_session() as session:
+        session.http_proxy.create(
```

Comment:
I feel we should check the default value of `content_default_http_proxy` too to keep a check if default value gets changed in future.

Reply:
I think its part of unittest already, or it can be covered in airgun, so non-blocking for this PR